src/PhpDoc/Tag/TypeAliasTag.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDoc\Tag;
+
+class TypeAliasTag
+{
+
+	private string $alias;
+
+	private string $type;
+
+	public function __construct(string $alias, string $type)
+	{
+		$this->alias = $alias;
+		$this->type = $type;
+	}
+
+	public function getAlias(): string
+	{
+		return $this->alias;
+	}
+
+	public function getType(): string
+	{
+		return $this->type;
+	}
+
+	/**
+	 * @param mixed[] $properties
+	 * @return TypeAliasTag
+	 */
+	public static function __set_state(array $properties): self
```

Comment:
This is not needed, objects of classes in `PHPStan\PhpDoc\Tag` are no longer saved in cache.

Reply:
fixed in [a9c6e17e](https://github.com/phpstan/phpstan-src/pull/460/commits/a9c6e17e9fdbb3d36c98587055c39bf9431b579f)